[docs] clarify that server load should be used in example and isn't optional (#8193)
sveltejs/kit: documentation/docs/20-core-concepts/20-load.md

Thanks to the generated `$types` module, we get full type safety.

> A `load` function in a `+page.js` file runs both on the server and in the browser. If your `load` function should _always_ run on the server (because it uses private environment variables, for example, or accesses a database) then you should put it in a `+page.server.js` instead.

Reviewer: A shorter version
```suggestion
A `load` function in a `+page.js` file runs both on the server and in the browser. If your `load` function should _always_ run on the server (because it uses private environment variables, for example, or accesses a database) then put it in a `+page.server.js` instead.
```

> A `load` function in a `+page.js` file runs both on the server and in the browser. If your `load` function should _always_ run on the server (because it uses private environment variables, for example, or accesses a database) then you should put it in a `+page.server.js` instead.

Reviewer 2: the sentence sounds funny with "should" used twice
```suggestion
A `load` function in a `+page.js` file runs both on the server and in the browser. If your `load` function should _always_ run on the server (because it uses private environment variables, for example, or accesses a database) then it would go in a `+page.server.js` instead.
```
